server/routes/comment.route.js

```diff
@@ -1,2 +1,35 @@
-// TODO: need to do comment vote routes here otherwise post will be
-// preloaded
+import express from 'express';
+import expressJwt from 'express-jwt';
+import transferField from '../middleware/transferField';
+import commentCtrl from '../controllers/comment.controller';
+import voteCtrl from '../controllers/vote.controller';
+import config from '../../config/config';
+
+const router = express.Router(); // eslint-disable-line new-cap
+
+
+router.route('/:commentId/upvote')
+  .post(expressJwt({ secret: config.jwtSecret })
+  // TODO: refactor to have these into one call like upvote: [method1, method2,...]
+  // upvoteHandlers
+  , transferField({source: 'comment', target: 'entity'})
```

Comment:
Why are you transferring the field here? This is a pretty big code smell. Generics and Magic Strings.